### ISSUE TITLE
FizzBuzzEnterpriseEdition does not run on my 1914 Radio.

### ISSUE THREAD

**Issue by DailyEndermite** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#625]
I have saw other issues (and solutions) to fixing this program on less modern OS like toasters and water bottles, but the fixes have not worked for my radio. It is equipped with "modern" computer parts, but it is very outdated and I am financially struggling so I am unable to afford a device that can run this program.

**Comment by Tylersuard** [comment]
DailyEndermite, the first commercially available radio came out in the 1920’s. Could you possibly be referring to the WWI “Horse-Pack” radios?

**Comment by DailyEndermite** [comment]
Yes, I meant to refer to the Horse Pack radios that were mainly used in WW1.
